typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -53,9 +53,39 @@ enum ViewSide {
     Top,
 }
 
+function MakeExpLabel(a: number): string {
+    return a.toExponential(2).replace("e", "E");
+}
+
+function UseExponential(a: number, b: number): [boolean, boolean, number] {
```

Comment:
I'm not a fan to rewrite the wheel.
Can't we use d3 format, or some standard lib ?